rxandroidble/src/main/java/com/polidea/rxandroidble2/internal/operations/LegacyScanOperation.java

```diff
@@ -48,6 +60,9 @@ public void onLeScan(BluetoothDevice device, int rssi, byte[] scanRecord) {
 
     @Override
     boolean startScan(RxBleAdapterWrapper rxBleAdapterWrapper, BluetoothAdapter.LeScanCallback scanCallback) {
+        if (this.filterUuids == null) {
+            RxBleLog.d("No library side filtering —> debug logs of scanned devices disabled");
```

Comment:
`RxBleLog.isAtLeast(LogConstants.DEBUG)`?

Reply:
It is not needed as there is no object creation involved